### ISSUE TITLE
Few Vercel error popup messages

### ISSUE THREAD

**Issue by aaadriana** [fac22/FACTube#88]
- Cannot read properties of null (reading 'id')
- Invalid input syntax for type uuid: "0"

**Comment by Moggach** [comment]
Yep, we think this is to do with the component being rendered even when user not logged in. Thank you 😊
